app/controllers/issueMapController.js

```diff
@@ -1,7 +1,133 @@
+import IssueMapAccessor from "../databaseAccessors/issueMapAccessor.js";
+import {
+  ErrorInvalidRequestBody,
+  ErrorUnexpected,
+  HttpError,
+  ErrorSectionNotFound,
+  ErrorIssueMapNotFound,
+} from "../error/errors.js";
+import ArticleStatus from "../models/enums/articleStatus.js";
+import DesignStatus from "../models/enums/designStatus.js";
+import PhotographyStatus from "../models/enums/photographyStatus.js";
+import WritingStatus from "../models/enums/writingStatus.js";
+import Article from "../models/dbModels/article.js";
+import IssueMap from "../models/dbModels/article.js";
+import ArticlesAccessor from "../databaseAccessors/articleAccessor.js";
+import UsersAccessor from "../databaseAccessors/userAccessor.js";
+
 /**
  * IssueMapController Class
  *
  * This class controls the behaviour of any web request
  * related to IssueMaps.
  */
-export default class IssueMapController {}
+
+export default class IssueMapController {
+  /**
+   * method to create and add an article from the issue map.
+   *
+   * Create and add an article from the issue map.
+   *
+   * @param {Request} req
+   * @param {Response} res
+   */
+  static async addAndCreateArticle(req, res) {
+    try {
+      console.log("Request Body:", req.body);
```

Comment:
Console logs are useful for debugging - just be sure to remove them (except from test files) before you commit your code